app/views/hosts/edit.html.erb

```diff
@@ -86,7 +86,9 @@
             <label for="orgName"><%= t '.org_name' %></label>
             <input type="text" class="form-control"
                    id="orgName"
-                   ng-model="host.org_name">
+                   ng-class="{true: 'error'}[submitted[stepIndex] && !host.org_name]"
```

Comment:
@opinyo did you test this? i think you need to add `name="org_name"` for the validation to kick in. you will see red around the input

Reply:
It works. Should I change it anyway?

Reply:
yes, please add for consistency's sake

Reply:
Done.